nemo/collections/asr/models/hybrid_rnnt_ctc_bpe_models.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+from typing import Dict, List, Optional, Union
+
+from omegaconf import DictConfig, ListConfig, OmegaConf, open_dict
+from pytorch_lightning import Trainer
+
+from nemo.collections.asr.metrics.wer_bpe import WERBPE, CTCBPEDecoding, CTCBPEDecodingConfig
+from nemo.collections.asr.models.rnnt_models import EncDecRNNTBPEModel
+from nemo.utils import logging, model_utils
+
+
+class EncDecHybridRNNTCTCBPEModel(EncDecRNNTBPEModel):
```

Comment:
Shouldnt this class inherit `EncDecHybridRNNTCTCModel` instead of the basic RNNT one?

Reply:
Also need to implement the data loader for subword in this class (similar to how it was done for EncDecRNNTBPEModel, probably can just copy paste that).

Reply:
Basically im thinking inheritance hierarchy will be : 

ASRModel -> EncDecCTCModel -> EncDecCTCBPEModel
                  -> EncDecRNNTModel -> EncDecRNNTBPEModel
                  -> EncDecRNNTCTCModel -> EncDecRNNTCTCBPEModel

Reply:
Extending the EncDecHybridRNNTCTCBPEModel from EncDecHybridRNNTCTCModel  would make to copy most of the EncDecRNNTModel here while by inhering from EncDecRNNTBPEModel, there is no need for them. This class is an extention of both EncDecRNNTBPEModel and EncDecHybridRNNTCTCModel  but extending from EncDecRNNTBPEModel makes it nicer.
EncDecRNNTCTCModel is not a type of EncDecRNNTBPEModel.